django/contrib/admin/templates/admin/nav_sidebar.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-<button class="sticky toggle-nav-sidebar" id="toggle-nav-sidebar" aria-label="{% translate 'Toggle navigation' %}"></button>
+<span class="sticky toggle-nav-sidebar" id="toggle-nav-sidebar" aria-label="{% translate 'Toggle navigation' %}"></span>
```

Comment:
I mentioned on the ticket that we shouldn't fix the border issue. If we did, the right fix is not to change it to a non-semantic element. This needs to be a button not only to be semantic but also to make it keyboard-accessible. It's true we could add tabindex but it still loses the semantic nature.

If you can fix this while keeping the `:focus` state when the user is tabbed on the element, and without changing it to a span, it might be okay.

Reply:
add outline: none; to fix it

Reply:
Thanks, but this makes it very hard to see when the toggle is focused (e.g. when tabbing). Because Chrome keeps the focused state after click, and until you click something else, I don't think it's reasonably possible to fix it.

Reply:
change to a to fix, focus it by tab key will show border, after click donnot have border

Reply:
I agree this should remain a button. Semantically, it is the most appropriate element, that is it provides custom user interaction. This element is not an anchor/link (no href) and a span isn't right for the reasons @knyghty pointed out.

Reply:
rollback to button and add blur() when mouseup